tox.ini

```diff
@@ -1,19 +1,30 @@
 [tox]
 minversion=2.3.1
 envlist =
-    py{27,35}-ansible22-{pylint,unit,flake8,yamllint,generate_validation}
+    py{27,35}-ansible22-lint
+    py{27,35}-ansible22-unit
 skipsdist=True
 skip_missing_interpreters=True
 
 [testenv]
 deps =
+    wheel
     -rtest-requirements.txt
-    py35-flake8: flake8-bugbear
-    ansible22: ansible~=2.2
+    lint: -rtest-requirements-lint.txt
+    unit: -rtest-requirements-unit.txt
+    py35-lint: flake8-bugbear
+    ansible22: ansible==2.2.0.0
 
 commands =
-    flake8: flake8
-    pylint: python setup.py lint
-    yamllint: python setup.py yamllint
+    lint: flake8
+    lint: python setup.py lint
+    lint: python setup.py yamllint
+    lint: python setup.py generate_validation
+    # TODO(rhcarvalho): check syntax of other important entrypoint playbooks
+    lint: ansible-playbook --syntax-check playbooks/byo/config.yml
     unit: nosetests
-    generate_validation: python setup.py generate_validation
+
+[travis:env]
+TEST_TYPE =
+    lint: lint
+    unit: unit
```

Comment:
Aha, thanks!

Would you mind me being paranoid and ask us to see a FAILED BUILD (on purpose), just to be sure that we will get build failures, with the expected messages, when there is a error?

After reading http://tox-travis.readthedocs.io/en/stable/toxenv.html#unignore-outcomes, I remember I've seen repos with CI always green even when it should be red.
The transition red->green is a lot more meaningful than seeing all green.

We can force a RED build by adding a dummy commit with mistakes, and then dropping it before merge.